src/Data/Uniques/Special/Generated.lua

```diff
@@ -762,6 +698,75 @@ function buildForbidden(classNotables)
 	table.insert(data.uniques.generated, table.concat(forbidden["Flesh"], "\n"))
 end
 
+function buildKeystoneItems(keystoneMap)
+	local skinOfTheLords = {
+		"Skin of the Lords",
+		"Simple Robe",
+		"League: Breach",
+		"Source: Upgraded from unique{Skin of the Loyal} using currency{Blessing of Chayula}",
+	}
+	local excludedPassiveKeystones = {
+		"Chaos Inoculation", -- to prevent infinite loop
+		"Necromantic Aegis", -- to prevent infinite loop
+	}
+
+	-- Keystones added by jewels don't have a position set in the tree data
+	local isKeystoneNative = function(node) return node.isKeystone and not node.isBlighted and node.x ~= nil end
+
+	local skinOfTheLordsKeystones = {}
+	local seen = {}
+	for _, node in pairs(keystoneMap) do		
+		if isKeystoneNative(node) and not isValueInArray(excludedPassiveKeystones, node.name) and not seen[node] then
+			table.insert(skinOfTheLordsKeystones, node.name)
+			seen[node] = true
```

Comment:
Did you find that there were duplicates when testing or is this just for future proofing?

Reply:
Answered my own question when checking out the branch...wonder why everything is in there twice...

Reply:
`ModParser` lowercases Impossible Escape's mod line before parsing it, so the second, lowercased key, was added to support accesses from the jewel data.